### DISCUSSION TITLE
Chromebook : Keepassxc browser doesn´t connect the Keepassxc DB in the native OEM Chrome browser.

### DISCUSSION THREAD

**Discussion by kcnypmek** [keepassxreboot/keepassxc#11106] [Q&A] [answered]
I installed KeepassXC on my Chromebook trough Linux Flatpak with success and it works fine. The only issue I have is with the OEM Chrome browser of my chromebook, because the Chrome extension doesn´t connect  with the KeepassXC DB.
To test I installed a Linux Chrome Browser and if I use this version the Chrome KeepassXC browser extension loads the KeepassXC DB and than it works fine. Can I solve the problem on the native OEM Chrome browser?

**Comment by droidmonkey** [comment, marked as answer]
We don't support Chrome OS
